<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Import</class>
 <widget class="QDialog" name="Import">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Importar</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Direcci√≥n de la fuente documental </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEditSource"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSource">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="recursos.qrc">
         <normaloff>:/images/import.png</normaloff>:/images/import.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>No hacer copia de seguridad</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Carpeta donde se guardara una copia de seguridad</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="lineEditBackup"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBackup">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="recursos.qrc">
         <normaloff>:/images/black_drive_backup.png</normaloff>:/images/black_drive_backup.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Progreso</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAceptar">
       <property name="text">
        <string notr="true">Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
